sources/packages/backend/libs/services/src/notifications/notification/notification.model.ts

```diff
@@ -30,6 +30,13 @@ export interface MSFAACancellationNotification {
   userId: number;
 }
 
+export interface OfferingChangeRequestInProgressWithStudentNotification {
```

Comment:
Maybe adding `application` to the name will help differentiate it from the actual offering change